### PR TITLE
Update RPM commands for new 3.13 auto-publish

### PR DESCRIPTION
**Pull request by gerrod3**
fixes: #251

### CODE REVIEW

File: pulpcore/cli/rpm/distribution.py [hunk 1]
```diff
@@ -60,6 +60,7 @@ def distribution(ctx: click.Context, pulp_ctx: PulpContext, distribution_type: s
 update_options = [
     click.option("--base-path"),
     click.option("--publication"),
+    click.option("--repository", callback=_repository_callback),
```
Comment (mdellweg) [inline pulpcore/cli/rpm/distribution.py:63]: This is def more than just adjusted tests.

Comment (gerrod3) [reply pulpcore/cli/rpm/distribution.py:63]: Well the option was missing, so I added it.

Comment (mdellweg) [reply pulpcore/cli/rpm/distribution.py:63]: And it's needed for the autopublish operations?
Then we can reflect this as a feature in the changelog. Like "Adjusted autopublish to new version of ...".

File: CHANGES/251.bugfix [hunk 2]
```diff
@@ -0,0 +1 @@
+Updated RPM auto-publish tests for future versions >=3.13.0.dev
```
Comment (daviddavis) [inline CHANGES/251.bugfix:1]: The changelog is meant for end users so test changes don't need to be called out. I would probably just repurpose this for the repository option as @mdellweg suggests.

**Review by daviddavis — APPROVED** [review]

**Pull request merged**